### DISCUSSION TITLE
pimcore asset folder

### DISCUSSION THREAD

**Discussion by remyahalza** [pimcore/pimcore#7830] [Q&A / Support]
Is it possible to change the asset folder?

**Comment by brusch** [comment]
What exactly you want to achieve? 
You mean on the local file-system? Then yes, see https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Configuration.html#page_Pimcore-constants

**Reply by remyahalza** [reply]
It was working Thanks
